gradoop-store/gradoop-accumulo/src/main/java/org/gradoop/flink/io/impl/accumulo/AccumuloDataSink.java

```diff
@@ -67,10 +67,11 @@ public void write(
       new VertexOutputFormat(getAccumuloConfig().getAccumuloProperties()));
     graphCollection.getEdges().output(
       new EdgeOutputFormat(getAccumuloConfig().getAccumuloProperties()));
-//    graphCollection.getEdges().output(
-//      new EdgeOutOutputFormat(getAccumuloConfig().getAccumuloProperties()));
-//    graphCollection.getEdges().output(
-//      new EdgeInOutputFormat(getAccumuloConfig().getAccumuloProperties()));
+    //add Edge-in and edge-out, TBD.
```

Comment:
No such tbd comments in code. Copy it to the corresponding issue and delete these comments.